src/main/java/de/retest/ui/descriptors/idproviders/RetestIdProvider.java

```diff
@@ -9,8 +9,6 @@
  */
 public interface RetestIdProvider {
 
-	public static final String ID_PROVIDER_CONFIGURATION_PROPERTY = "de.retest.retestIdProvider";
```

Comment:
Same here: public API. (Though we have defined the property here and in `RetestIdProviderUtil`, which one should we keep?)